### DISCUSSION TITLE
Zookeeper admin port doesn't change after configuration update

### DISCUSSION THREAD

**Discussion by cvetan** [Homebrew/discussions#4663] [Everyday usage]
### Output of `brew config`

```shell
HOMEBREW_VERSION: 4.1.0
ORIGIN: https://github.com/Homebrew/brew
HEAD: ac93842f8d13913f462aedb246cfacd08d6b7f05
Last commit: 4 days ago
Core tap JSON: 23 Jul 10:32 UTC
HOMEBREW_PREFIX: /opt/homebrew
HOMEBREW_CASK_OPTS: []
HOMEBREW_MAKE_JOBS: 8
Homebrew Ruby: 2.6.10 => /System/Library/Frameworks/Ruby.framework/Versions/2.6/usr/bin/ruby
CPU: octa-core 64-bit arm_firestorm_icestorm
Clang: 14.0.0 build 1400
Git: 2.37.1 => /Library/Developer/CommandLineTools/usr/bin/git
Curl: 7.87.0 => /usr/bin/curl
macOS: 12.6.7-arm64
CLT: 14.2.0.0.1.1668646533
Xcode: N/A
Rosetta 2: false
```

### Output of `brew doctor`

```shell
Your system is ready to brew.
```

### Description of issue

When I change admin port in zookeeper.properties and run kafka again, seems like the port didn't change at all and 8080 is still ocupied. I am running java applications so I need 8080 free, but I also need zookeeper admin enabled. Am I doing something wrong, could it be changed on some other place? So I am using pretty much default configuration as homebrew installed it. Except this port I am trying to change.

**Comment by osalbahr** [comment]
How can I replicate the same error locally?

**Reply by cvetan** [reply]
Try running something on port 8080, it won't let you with default configuration.

**Comment by rasenderhase** [comment]
Try to add the following property in `/usr/local/etc/zookeeper/zoo.cfg`:
```
admin.enableServer=false
```

or if you need the admin server:
```
admin.serverPort=8088
```
